### DISCUSSION TITLE
Option to mute notifications

### DISCUSSION THREAD

**Discussion by timche** [zoidsh/meru#244] [Feature Requests] [answered]
Inspired by: https://github.com/sindresorhus/caprine#mute-desktop-notifications

**Comment by timche** [comment, marked as answer]
Added in [v3.0.0-alpha.15](https://github.com/timche/gmail-desktop/releases/tag/v3.0.0-alpha.15).

**Comment by chrisma** [comment]
I would especially appreciate muting/disabling the `download cancelled` notifications (e.g. when bringing up the save dialog, seeing that the file is already present and canceling). I know the download was cancelled in this case: I clicked the button.

![image](https://user-images.githubusercontent.com/1652117/125275325-2c2b9380-e30f-11eb-83e1-b020bb7c3351.png)

**Reply by timche** [reply]
Thanks for your feedback. I'll check if this can be made smarter.
